modules/installation/pages/upgrade.adoc

```diff
@@ -199,28 +186,75 @@ Welcome to the TigerGraph platform installer!
 [PROGRESS]: 08:20:33 Switching to new version 4.2.1 successfully...
 ----
 
-=== Managing Service Disruption
+=== Managing Service Disruption During Rolling Upgrade
 
-During the rolling upgrade, expect a temporary service disruption. If the upgrade fails or the connection is lost, user can *resume* the upgrade process by running:
+During a rolling upgrade, temporary service interruptions are expected. If the upgrade process fails or the connection is unexpectedly lost, you can resume the upgrade by running:
 
 [source.wrap,bash]
 ----
 $ gadmin upgrade --resume
 ----
 
-If the upgrade encounters an issue, the error prompt will guide you on how to continue the upgrade manually. 
-Use the following command if you need to manually finish the upgrade:
+=== Rolling Upgrade Failure with Error
```

Comment:
Change the title to something like "Recovering from An Incomplete Rolling Backup".
The subsection should be on Determining that Your Rolling Backup Didn't Finish.
The next section(s) should be on your two options: Continue, but as a manual, offline backup. Or, revert back to the old version. If the disruption is not expected to happen again, they could them retry.